ee/tabby-ui/lib/utils.ts

```diff
@@ -41,3 +41,18 @@ export function formatDate(input: string | number | Date): string {
     year: 'numeric'
   })
 }
+
```

Comment:
should be good to leave this function in `prompt-form.tsx` until there's a need for reuse.